code/swb_base/WeaponBase.Attack.cs

```diff
@@ -283,36 +279,10 @@ public virtual void ShootBullet(float spread, float force, float damage, float b
         var endPos = player.EyePosition + forward * 999999;
```

Comment:
The spread calculation should use a seed so that the random values are the same on client and server and then you can combine the bullet `FireSV/CL` functions

Reply:
I had left these untouched on purpose, as I didn't want to touch and potentially break more than I needed to